aries-site/src/examples/foundation/icons/IconPlainExample.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { Box } from 'grommet';
 // TODO replace with DS icon package when available
-import { Hpe, Hpi } from 'grommet-icons';
+import { Hpe, HpeLabs } from 'grommet-icons';
+import {
+  Achievement,
+  Aid,
+  Alarm,
+} from '@hpe-design/icons-grommet';
 
 export const IconPlainExample = () => (
   <Box direction="row-responsive" gap="medium">
-    <Hpe color="plain" size="large" />
-    <Hpi color="plain" size="large" />
+    <Achievement color="icon-weak" size="xxlarge" />
+    <Aid color="icon-default" size="xxlarge" />
+    <Alarm color="icon-strong" size="xxlarge" />
+    <Hpe color="plain" size="xxlarge" />
+    <HpeLabs color="plain" size="xxlarge" />
```

Comment:
why did you change this from `Hpi` to `HpeLabs` do we need either can we just do `Hpe`?

Reply:
I believe this is done @britt6612. Please verify and resolve if you are happy.

Reply:
yes all good here will resolve.